packages/css/src/field.css

```diff
@@ -130,7 +130,7 @@
     }
   }
 
-  & .ds-input {
+  & :has(.ds-input) {
```

Comment:
Det blir ikke helt rett? :o dette endrer hvilket element som styles - vi kan ringes om du vil ☺️

Reply:
Nei, det er ikke det. Driver å tukler med det akkuratt nå. Må jo sette den til review for å få chromatic visuelle tester 🙊